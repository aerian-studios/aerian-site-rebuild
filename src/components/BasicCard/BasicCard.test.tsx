import * as React from "react";

import * as renderer from "react-test-renderer";

import { BasicCard } from ".";

describe("BasicCard", () =>
    it("renders correctly", () => {
        const tree = renderer
            .create(
                <BasicCard>
                    <p>Hi</p>
                </BasicCard>
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    }));
