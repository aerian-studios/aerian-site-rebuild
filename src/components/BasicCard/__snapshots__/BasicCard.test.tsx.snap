// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasicCard renders correctly 1`] = `
<div
  className="component"
>
  <p>
    Hi
  </p>
</div>
`;
